README.md

```diff
@@ -27,7 +27,7 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove the FAQ section from your readme file if you don't have anything to put in this section to make your readme file clean and descriptive 👍